Pod/Classes/CDTQQueryValidator.m

```diff
@@ -287,28 +349,46 @@ + (BOOL)validateClause:(NSDictionary *)clause
     //$exits lt
 
     NSArray *validOperators =
-        @[ @"$eq", @"$lt", @"$gt", @"$exists", @"$not", @"$ne", @"$gte", @"$lte" ];
+        @[ @"$eq", @"$lt", @"$gt", @"$exists", @"$not", @"$gte", @"$lte", @"$in" ];
 
     if ([clause count] == 1) {
         NSString *operator= [clause allKeys][0];
 
-    if ([validOperators containsObject:operator]) {
-        // contains correct operator
-        id clauseOperand = [clause objectForKey:[clause allKeys][0]];
-        // handle special case, $notis the only op that expects a dict
-        if ([operator isEqualToString:@"$not"] && [clauseOperand isKindOfClass:[NSDictionary class]]) {
-            return [CDTQQueryValidator validateClause:clauseOperand];
-
-        } else if ([CDTQQueryValidator validatePredicateValue:clauseOperand
-                                                  forOperator:[clause allKeys][0]]) {
-            return YES;
+        if ([validOperators containsObject:operator]) {
+            // contains correct operator
+            id clauseOperand = [clause objectForKey:[clause allKeys][0]];
+            // handle special case, $notis the only op that expects a dict
+            if ([operator isEqualToString:NOT]) {
+                return [clauseOperand isKindOfClass:[NSDictionary class]] &&
+                       [CDTQQueryValidator validateClause:clauseOperand];
+
+            } else if ([operator isEqualToString:IN]) {
+                return [clauseOperand isKindOfClass:[NSArray class]] &&
+                       [CDTQQueryValidator validateInListValues:clauseOperand];
```

Comment:
The naming with `InListValues` is confusing, "is _something_ in `ListValues`" it sounds like and I keep double-taking. Can this just be `validateListValues`? There are other `*In*` methods which also read confusingly.